### PR TITLE
Testing components

### PR DESCRIPTION
**Pull request by sawood14012**
added default test cases for Box and Button components.
run `npm test` or `yarn test` to check
and use the same version of react dom , react  and react test renderer 

`"react": "16.8.3",
"react-dom": "16.8.3",
"react-test-renderer": "16.8.3"`

this test will produce files in __snapshots__ folder like box.test.js.snap with the compiled result.

### CODE REVIEW

**Comment by sawood14012** [comment]
I will update all the tests soon.. :sunglasses:

**Comment by piyush97** [comment]
#41 Resolved Partially

**Comment by piyush97** [comment]
@all-contributors add @sawood14012 	as tester

**Comment by piyush97** [comment]
@all-contributors add @sawood14012 	as userTesting

**Review by piyush97 — CHANGES_REQUESTED** [review]
Bhai please remove the appropriate comment lines. :)

**Review by piyush97 — CHANGES_REQUESTED** [review]
@sawood14012 	Please remove the comments

File: src/components/Media/__snapshots__/Media.test.js.snap [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (piyush97) [inline src/components/Media/__snapshots__/Media.test.js.snap:1]: Comments like these

**Review by piyush97 — APPROVED** [review]

**Pull request merged**